src/array.jl

```diff
@@ -212,6 +212,13 @@ end
 
             convert($A{T, N, R}, A)
         end
+
+        function vcat(A::$A...)
+            L, O = sortedmerge(map(levels, A)...)
```

Comment:
You're absolutely right, I think I inherited this from DataArrays, NullableArrays or Base. But I think I would prefer if `x` was used everywhere. :-)